dashboard/web/locales/en/neighbor.js

```diff
@@ -0,0 +1,16 @@
+export default {
+  TITLE: 'Neighbors',
+  WARNING: 'Warning: No entry to the neighbor. Or the outside will not come in.',
```

Comment:
Warning: Local node has no inbound neighbor, it's possible that local node is unreachable from outside, e.g. behind firewall or NAT.